zookeeper-docs/src/main/resources/markdown/zookeeperStarted.md

```diff
@@ -166,7 +166,7 @@ From the shell, type `help` to get a listing of commands that can be executed fr
         addauth scheme auth
         delete path [version]
         deleteall path
-        setquota -n|-b val path
+        setquota [-n|-b] val path
```

Comment:
I don't believe this is optional, see SetQuotaCommand, in particular "setRequired(true)"

```
    public SetQuotaCommand() {
        super("setquota", "-n|-b val path");
        
        OptionGroup og1 = new OptionGroup();
        og1.addOption(new Option("b", true, "bytes quota"));
        og1.addOption(new Option("n", true, "num quota"));
        og1.setRequired(true);
        options.addOptionGroup(og1);
   }
```

Reply:
`OptionGroup.setRequired(true)` means that: this `OptionGroup` is mandatory and this two Options (`-n;-b`) is exclusive

Reply:
ok,thanks.